### PR TITLE
minor grammar change

### PR DESCRIPTION
**Pull request by taylor**
suggested by @iawells to change

`network functionality, which is developed` to `network functionality and is developed`

### CODE REVIEW

File: doc/glossary.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 **Cloud Native** - TBD
 
-**Cloud Native Network Function** - A cloud native network function (CNF) is a [cloud native application](https://github.com/cncf/glossary/blob/main/definitions/cloud_native_apps.md) that provides network functionality, which is developed using cloud native principles. 
+**Cloud Native Network Function** - A cloud native network function (CNF) is a [cloud native application](https://github.com/cncf/glossary/blob/main/definitions/cloud_native_apps.md) that provides network functionality and is developed using cloud native principles.
```
Comment (lixuna) [inline doc/glossary.md]:
```suggestion
**Cloud Native Network Function** - A cloud native network function (CNF) is a [cloud native application](https://github.com/cncf/glossary/blob/main/definitions/cloud_native_apps.md) that provides network functionality and is developed using cloud native principles.
```

Comment (lixuna) [reply doc/glossary.md]: remove trailing space to resolve linter error

`9:267 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]`

**Review by lixuna — CHANGES_REQUESTED** [review]

**Review by electrocucaracha — APPROVED** [review]
/lgtm

**Review by lixuna — APPROVED** [review]
LGTM

**Pull request merged**